projects/arlas-toolkit/assets/i18n/es.json

```diff
@@ -82,11 +97,10 @@
 	"Send password reset email": "Recibir un correo electrónico de restablecimiento",
 	"Already have an account?": "¿Ya tiene una cuenta?",
 	"Log In": "Iniciar sesión",
-	"interval:": "Intervalo",
-	"x-axis:": "Abscisa :",
-	"From": "Desde",
-	"": "",
-	"y-axis:": "Ordenada :",
+	"x-axis": "",
```

Comment:
Is it normal they don't have translations ? Is "x-axis:" still in the file ?

Reply:
I'd check in another PR, sorry